opentelemetry-sdk/src/opentelemetry/sdk/_metrics/measurement_consumer.py

```diff
@@ -0,0 +1,29 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from abc import ABC, abstractmethod
+
+from opentelemetry.sdk._metrics.measurement import Measurement
+
+
+class MeasurementConsumer(ABC):
+    @abstractmethod
+    def consume(self, measurement: Measurement):
+        pass
+
+
+class SerialMeasurementConsumer(MeasurementConsumer):
```

Comment:
Can we call it DefaultMeasurementConsumer so folks who read the doc will know what they're looking at

Reply:
I think it is better to fix the doc. We have discussed before that DefaultX is not a good name.

Reply:
What does `Serial` mean?

Reply:
> We have discussed before that DefaultX is not a good name.

I thought that was in regards to the no-op implementations. I can see it is an overloaded term in our codebase, so am ok with a different name. However I am also confused why this is `Serial`. Maybe `Synchronous` would be more appropriate?

Reply:
All right, renamed to `Synchronous` :v: